<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkWindow</class>
 <widget class="QDialog" name="networkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="15,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>801</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,4,1">
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="geometry">
               <rect>
                <x>16</x>
                <y>140</y>
                <width>251</width>
                <height>29</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
background: white;

border-radius: 10px;
padding: 5px;
}</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>110</y>
                <width>61</width>
                <height>30</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>User 1</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QFrame" name="frame_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>113</width>
                <height>297</height>
               </rect>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>140</y>
                <width>251</width>
                <height>29</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
background: white;

border-radius: 10px;
padding: 5px;
}</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>110</y>
                <width>61</width>
                <height>30</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>User 2</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>301</y>
          <width>781</width>
          <height>241</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0.01, x2: 0, y2: 0.5, stop: 0 rgb(230, 230, 255), stop: 1 rgb(0, 170, 255));
border-style: solid;
border-color: black;
border-radius: 10px;
color: white;
}
QPushButton::hover{
background:rgba(0, 170, 255, 200);

}
QPushButton::pressed{
background: rgba(0, 170, 255, 200);

}</string>
         </property>
         <property name="text">
          <string>Most Active User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_16">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0.01, x2: 0, y2: 0.5, stop: 0 rgb(230, 230, 255), stop: 1 rgb(0, 170, 255));
border-style: solid;
border-color: black;
border-radius: 10px;
color: white;
}
QPushButton::hover{
background:rgba(0, 170, 255, 200);

}
QPushButton::pressed{
background: rgba(0, 170, 255, 200);

}</string>
         </property>
         <property name="text">
          <string>Most Influencer User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_17">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0.01, x2: 0, y2: 0.5, stop: 0 rgb(230, 230, 255), stop: 1 rgb(0, 170, 255));
border-style: solid;
border-color: black;
border-radius: 10px;
color: white;
}
QPushButton::hover{
background:rgba(0, 170, 255, 200);

}
QPushButton::pressed{
background: rgba(0, 170, 255, 200);

}</string>
         </property>
         <property name="text">
          <string>Follow Suggestions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
           <hintingpreference>PreferDefaultHinting</hintingpreference>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: qlineargradient(x1: 0, y1: 0.01, x2: 0, y2: 0.5, stop: 0 rgb(230, 230, 255), stop: 1 rgb(0, 170, 255));
border-style: solid;
border-color: black;
border-radius: 10px;
color: white;
}
QPushButton::hover{
background:rgba(0, 170, 255, 200);

}
QPushButton::pressed{
background: rgba(0, 170, 255, 200);

}</string>
         </property>
         <property name="text">
          <string>Mutual Followers</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
